test/index.spec.js

```diff
@@ -59,6 +59,8 @@ const utils = {
   },
 }
 
+const normalizeChunkNames = (source) => source.replaceAll(/\/chunks\/\d+\.js/g, '/chunks/CHUNK_ID.js')
```

Comment:
The chunk id is too easy to break between platforms, and shouldn't indicate a test failure. This normalises them all to "CHUNK_ID"